src/decima/__init__.py

```diff
@@ -1,4 +1,6 @@
 import sys
```

Comment:
Why is it necessary to import predict_save_attributions here?

Reply:
there are the main functionalities of the package so should be importable with `from decima import predict_save_attributions`. Also, `__all__` added now.